app/main/forms/auth_forms.py

```diff
@@ -1,38 +1,10 @@
 from flask_wtf import Form
-from wtforms import PasswordField
-from wtforms.validators import DataRequired, Regexp, EqualTo, Length
+from wtforms.validators import DataRequired
```

Comment:
By and large we use `InputRequired` now, so it might be good to switch over to using that everywhere so it's consistent across this app. Should be a simple find/replace, no other changes needed.

The reason for this is that DataRequired does some extra conversions on any submitted data and checks that it's Truth-y, but as part of doing this is can sometimes throw an error message which isn't overridden properly by the message you pass into the constructor. With InputRequired this doesn't happen.

Reply:
Nice